api/app/signals/apps/reporting/models/export.py

```diff
@@ -1,10 +1,12 @@
 # SPDX-License-Identifier: MPL-2.0
-# Copyright (C) 2019 - 2021 Gemeente Amsterdam
+# Copyright (C) 2019 - 2022 Gemeente Amsterdam
 from django.contrib.gis.db import models
 
-from signals.apps.reporting.utils import _get_storage_backend
+# TODO: Can this be removed?
+# from signals.apps.reporting.utils import _get_storage_backend
 
-storage = _get_storage_backend(using='horeca')
+# storage = _get_storage_backend(using='horeca')
+storage = None
```

Comment:
This will break the current Swift storage. I would propose to remove the code if the export is not used or to make it compatible with Azure storage. For Swift storage the configuration still exists.

Reply:
Discussed this, it is ok to break the current export.

Reply:
At this moment this export is not used anywhere.
If this is correct the code will be removed OR if not correct this will be fixed in a next PR.